lms/static/js/learner_dashboard/models/course_enroll_model.js

```diff
@@ -0,0 +1,97 @@
+/**
+ *  Track optIn status and submit form
+ */
+;(function (define) {
+    'use strict';
+
+    define([
+            'backbone',
+            'jquery',
+            'underscore'
+        ],
+        function( Backbone, $, _ ) {
+            return Backbone.Model.extend({
+                defaults: {
+                    course_id: '',
+                    optIn: false,
+                    enrolled: false,
+                },
+
+                initialize: function( attributes, options ) {
+                    var api = '/api/commerce/v0/baskets/';
+
+                    this.set({
+                        course_id: options.courseId,
+                    });
+
+                    this.url = api;
```

Comment:
The url is coming from a hardcoded var so it makes sense to define it via ```urlRoot``` http://backbonejs.org/#Model-urlRoot